test/evergreen.yml

```diff
@@ -1384,6 +1384,37 @@ tasks:
               ./t -1 -C "direct_io=[data]" -c ../../../test/format/CONFIG.stress
             done
 
+  - name: wiredtiger
```

Comment:
The legacy name `wiredtiger` does not make much sense when migrating to Evergreen. I'd prefer a task name indicating the test purpose/area. In this case, the job runs `make check` to test a few different configurations.